packages/sdk/src/fetcher/core/index.ts

```diff
@@ -83,60 +51,70 @@ export class CoreFetcher implements ICoreFetcher {
         );
         if (missingTokens.length === 0) return cachedTokens;
 
-        const multicall = getContract({
-            address: CHAIN_ADDRESSES[chainId].multicall,
-            abi: MULTICALL_ABI,
-            publicClient: publicClient,
+        const result = await publicClient.multicall({
+            contracts: addresses.flatMap((address) => [
+                { address, abi: ERC20_ABI, functionName: "name" },
+                {
+                    address,
+                    abi: ERC20_ABI,
+                    functionName: "symbol",
+                },
+                {
+                    address,
+                    abi: ERC20_ABI,
+                    functionName: "decimals",
+                },
+                {
+                    address,
+                    abi: BYTES_NAME_ERC20_ABI,
+                    functionName: "name",
+                },
+                {
+                    address,
+                    abi: BYTES_SYMBOL_ERC20_ABI,
+                    functionName: "symbol",
+                },
+            ]),
+            allowFailure: true,
         });
-
-        const calls = addresses.flatMap((target) => [
-            { target, callData: ERC20_NAME_FUNCTION_DATA },
-            { target, callData: ERC20_SYMBOL_FUNCTION_DATA },
-            { target, callData: ERC20_DECIMALS_FUNCTION_DATA },
-            { target, callData: ERC20_BYTES_NAME_FUNCTION_DATA },
-            { target, callData: ERC20_BYTES_SYMBOL_FUNCTION_DATA },
-        ]);
-
-        const { result } = await multicall.simulate.tryAggregate([
-            false,
-            calls,
-        ]);
+        // .simulate.tryAggregate([
```

Comment:
We can remove this I guess.

Reply:
Addressed in 6e91b608ec812067b109c538412aaf1a22661e48